staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -702,6 +727,10 @@ message WindowsContainerSecurityContext {
 
     // The contents of the GMSA credential spec to use to run this container.
     string credential_spec = 2;
+
+    // If set runtime must run container as a 'host process' container; 
+    // otherwise, the runtime MUST return error.
```

Comment:
The phrasing on this comment sounds a little awkward. I think it means error should be returned if "host process" containers are not supported, but it sounds like it will error if the field is `false`.